index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <main class="container">
+        
+        <header class="header-container">
+            <div class="logo-container"></div>
+            <div class="navBar-container">
+
+            </div>
+
+        </header>
+            
```

Comment:
should we add an article or div for the dropdowns to live in? 

`<article class="locationSelect-dropdowns"`